app/homepage.py

```diff
@@ -14,73 +14,131 @@
+def build_map_style_button():
+
+    button = html.Button(children='Afficher les niveaux de risques',
+                         id='map_style_button')
+
+    return html.Center(button)
+
+
+# This function takes as input the number of clicks on the button defined above and returns the layer style to use
+def choose_map_style(n_clicks):
+
+    # Because we start with the alerts map, if the number of clicks is even, this means that
+    # we are still using the alerts map and we may want to risk one
+    if n_clicks % 2 == 0:
+        button_content_map = 'Afficher les niveaux de risques'
+        map_object = build_alerts_map()
+        slider = ' '
+
+    # If the number of clicks is odd, this means we are using the risks map and may
+    # want to come back to alerts one
+    else:
+        button_content_map = 'Revenir à la carte Alertes'
+        map_object = build_risks_map()
+        slider = build_opacity_slider()
+
+    return button_content_map, map_object, slider
+
+
+# This function either displays or hides meteo graphs from graphs.py
+def meteo_graphs(display=False):
+    if display is True:
+        return dbc.Row(
+            [
+                dbc.Col([html.H2("Données météorologiques"), dcc.Graph(figure=meteo_fig)],
+                        md=4),
+                dbc.Col([html.H2("another indicator"),
+                        dcc.Graph(figure={"data": [{"x": [1, 2, 3], "y": [1, 4, 9]}]})],
+                        md=4),
+                dbc.Col([html.H2("a third indicator"),
+                        dcc.Graph(figure={"data": [{"x": [1, 2, 3], "y": [1, 4, 9]}]})],
+                        md=4),
+            ])
+    else:
+        return ''
+
+
+# ------------------------------------------------------------------------------
+# Content and App layout
 
-# instantiating the map object from alerts.py: to be modified depending on
-# what we expect the user to view first on homepage
+# Instantiating the alert map object from alerts.py and setting it a the default map object:
 map_object = build_alerts_map()
 
-# instantiating fig objects from graphs.py functions
+# Instantiating fig objects from graphs.py functions
 meteo_fig = generate_meteo_fig()
 
+# Instantiating navbar object from navbar.py
 nav = Navbar()
 
-body = dbc.Container(
-    [dbc.Row(
-        [dbc.Col(html.H1('Bienvenue sur Pyronear')),
-         dbc.Col(dcc.Dropdown(id='user_department_input',
-                              options=[{'label': 'Ardèche', 'value': 'Ardèche'},
-                                       {'label': 'Gard', 'value': 'Gard'},
-                                       {'label': 'Landes', 'value': 'Landes'}
-                                       ]))
+# Instantiating map layers button object from alters.py
+map_layers_button = build_layer_style_button()
+
+# Instantiating map style button object
+map_style_button = build_map_style_button()
+
+# Instantiating meteo graphs, set to True to display them under the map, Fale to hide them
```

Comment:
```suggestion
# Instantiating meteo graphs, set to True to display them under the map, False to hide them
```

Reply:
Thanks, I will fix all this missprints in the next commit